tests/test_zeyple.py

```diff
@@ -113,8 +108,8 @@ def test_process_message_with_multipart_message(self):
             --=_504b4162.Gyt30puFsMOHWjpCATT1XRbWoYI1iR/sT4UX78zEEMJbxu+h--
         """), ["torvalds@linux-foundation.org"])
 
-        self.assertIsNotNone(emails[0]['X-Zeyple'])
-        self.assertTrue(emails[0].is_multipart())
+        assert emails[0]['X-Zeyple'] is not None
+        assert emails[0].is_multipart() is not None
```

Comment:
Any reason why you added `is not None`? Doesn't seem to make sense since `is_multipart()` returns a boolean.